src/views/preview/embed-view.jsx

```diff
@@ -0,0 +1,92 @@
+// embed view
+
+const bindAll = require('lodash.bindall');
+const React = require('react');
+const PropTypes = require('prop-types');
+const injectIntl = require('react-intl').injectIntl;
+
+const ProjectViewHOC = require('./project-view-hoc.jsx');
+const ErrorBoundary = require('../../components/errorboundary/errorboundary.jsx');
+const projectShape = require('./projectshape.jsx').projectShape;
+const NotAvailable = require('../../components/not-available/not-available.jsx');
+const Meta = require('./meta.jsx');
+
+const GUI = require('scratch-gui');
+const IntlGUI = injectIntl(GUI.default);
+
+const Sentry = require('@sentry/browser');
```

Comment:
If you do that, don't forget to remove it from here.

Reply:
Make sure you reload the GH page -- this the code was already removed